File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/C/PRAK601-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main() {
    int V,H;

    printf("Masukkan ukuran Vertikal (V) dan Horizontal (H) matriks : ");
    scanf("%d %d", &V,&H);
    printf("Masukkan angka matriks : ");

    int arr[V][H];
        
        for(int i = 0 ; i < V ; i++) {
            for(int j = 0 ; j < H ; j++) {
                scanf("%d", &arr[i][j]);
            }
        }

        printf("\nMatriks %d x %d : \n\n",V,H);

        for(int i = 0 ; i < V ; i++) {
            for(int j = 0 ; j < H ; j++) {
                printf("%d ", arr[i][j]);
            }
            printf("\n");
        }
}

File: semester I/Praktikum Pemrograman I/Modul6_JovanGilbertNatamasindah_2310817310002/Python/PRAK601-2310817310002-JovanGilbertNatamasindah.py
import numpy as np

V, H = map(int, input("Masukkan ukuran Vertikal (V) dan Horizontal (H) matriks : ").split())

ELEMEN_ARRAY = list(map(int,input("Masukkan angka matriks : ").split(" ")))

print(f"\nMatriks {V} x {H} :\n")

arr = np.array(ELEMEN_ARRAY).reshape((V, H))

for HASIL in arr :
    print(*HASIL)   


